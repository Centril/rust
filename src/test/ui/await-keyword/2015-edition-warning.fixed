// run-rustfix

#![allow(non_camel_case_types)]
#![deny(keyword_idents)]

mod outer_mod {
    pub mod r#await {
//~^ ERROR `await` is a keyword
//~| WARN was previously accepted
//~| WARN hard error
        pub struct r#await;
//~^ ERROR `await` is a keyword
//~| WARN was previously accepted
//~| WARN hard error
    }
}
use outer_mod::r#await::r#await;
//~^ ERROR `await` is a keyword
//~| ERROR `await` is a keyword
//~| WARN was previously accepted
//~| WARN hard error
//~| WARN was previously accepted
//~| WARN hard error

fn main() {
    match r#await { r#await => {} }
//~^ ERROR `await` is a keyword
//~| ERROR `await` is a keyword
//~| WARN was previously accepted
//~| WARN hard error
//~| WARN was previously accepted
//~| WARN hard error
}
