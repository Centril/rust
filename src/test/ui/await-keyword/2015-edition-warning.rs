// run-rustfix

#![allow(non_camel_case_types)]
#![deny(keyword_idents)]

mod outer_mod {
    pub mod await {
//~^ ERROR `await` is a keyword
//~| WARN was previously accepted
//~| WARN hard error
        pub struct await;
//~^ ERROR `await` is a keyword
//~| WARN was previously accepted
//~| WARN hard error
    }
}
use outer_mod::await::await;
//~^ ERROR `await` is a keyword
//~| ERROR `await` is a keyword
//~| WARN was previously accepted
//~| WARN hard error
//~| WARN was previously accepted
//~| WARN hard error

fn main() {
    match await { await => {} }
//~^ ERROR `await` is a keyword
//~| ERROR `await` is a keyword
//~| WARN was previously accepted
//~| WARN hard error
//~| WARN was previously accepted
//~| WARN hard error
}
