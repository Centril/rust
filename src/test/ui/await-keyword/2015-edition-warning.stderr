error: `await` is a keyword in the 2018 edition
  --> $DIR/2015-edition-warning.rs:7:13
   |
LL |     pub mod await {
   |             ^^^^^ help: you can use a raw identifier to stay compatible: `r#await`
   |
note: lint level defined here
  --> $DIR/2015-edition-warning.rs:4:9
   |
LL | #![deny(keyword_idents)]
   |         ^^^^^^^^^^^^^^
   = warning: this was previously accepted by the compiler but is being phased out
   = warning: it will become a hard error in the 2018 edition!
   = note: for more information, see issue #49716 <https://github.com/rust-lang/rust/issues/49716>

error: `await` is a keyword in the 2018 edition
  --> $DIR/2015-edition-warning.rs:11:20
   |
LL |         pub struct await;
   |                    ^^^^^ help: you can use a raw identifier to stay compatible: `r#await`
   |
   = warning: this was previously accepted by the compiler but is being phased out
   = warning: it will become a hard error in the 2018 edition!
   = note: for more information, see issue #49716 <https://github.com/rust-lang/rust/issues/49716>

error: `await` is a keyword in the 2018 edition
  --> $DIR/2015-edition-warning.rs:17:16
   |
LL | use outer_mod::await::await;
   |                ^^^^^ help: you can use a raw identifier to stay compatible: `r#await`
   |
   = warning: this was previously accepted by the compiler but is being phased out
   = warning: it will become a hard error in the 2018 edition!
   = note: for more information, see issue #49716 <https://github.com/rust-lang/rust/issues/49716>

error: `await` is a keyword in the 2018 edition
  --> $DIR/2015-edition-warning.rs:17:23
   |
LL | use outer_mod::await::await;
   |                       ^^^^^ help: you can use a raw identifier to stay compatible: `r#await`
   |
   = warning: this was previously accepted by the compiler but is being phased out
   = warning: it will become a hard error in the 2018 edition!
   = note: for more information, see issue #49716 <https://github.com/rust-lang/rust/issues/49716>

error: `await` is a keyword in the 2018 edition
  --> $DIR/2015-edition-warning.rs:26:11
   |
LL |     match await { await => {} }
   |           ^^^^^ help: you can use a raw identifier to stay compatible: `r#await`
   |
   = warning: this was previously accepted by the compiler but is being phased out
   = warning: it will become a hard error in the 2018 edition!
   = note: for more information, see issue #49716 <https://github.com/rust-lang/rust/issues/49716>

error: `await` is a keyword in the 2018 edition
  --> $DIR/2015-edition-warning.rs:26:19
   |
LL |     match await { await => {} }
   |                   ^^^^^ help: you can use a raw identifier to stay compatible: `r#await`
   |
   = warning: this was previously accepted by the compiler but is being phased out
   = warning: it will become a hard error in the 2018 edition!
   = note: for more information, see issue #49716 <https://github.com/rust-lang/rust/issues/49716>

error: aborting due to 6 previous errors

